set.mm

```diff
@@ -320962,6 +320962,24 @@ need not be complete (for instance if the given set is infinite
       IUVPRUWBUVBMYJYLFYGUVOGYCUUEUVEUUCUWBUUDXKYBUWAXPUWDUWHUVOSZUUHXLXAUVMAFY
       NUVQUVNYKTKYMTKYNUVQRUVMFTYIYJUWEXMZUVMFTYIYLUWIXMZUVOYKYMUWJXNXCXOXQUVNY
       KYMUWKUWLXRXSWJXOWLUMXTVRYA $.
+
+    ${
+      rrxvsca.r $e |- .xb = ( .s ` H ) $.
+      rrxvsca.i $e |- ( ph -> I e. V ) $.
+      rrxvsca.j $e |- ( ph -> J e. I ) $.
+      rrxvsca.a $e |- ( ph -> A e. RR ) $.
+      rrxvsca.x $e |- ( ph -> X e. ( Base ` H ) ) $.
+      $( The scalar product over generalized Euclidean spaces is the component
+         wise real number multiplication.  (Contributed by Thierry Arnoux,
+         18-Jan-2023.) $)
+      rrxvsca $p |- ( ph -> ( ( A .xb X ) ` J ) = ( A x. ( X ` J ) ) ) $=
```

Comment:
That was seriously hard to understand. There is no scalar product here. "Generalized Euclidean space" is a red herring. It would help to have a sentence like:
```
In ` ( RR^ `` I ) ` , the ` J ` th component of a scalar multiple is the (same)
multiple of the ` J ` th component.
```
Add or replace, as you prefer.
(ping @tirix, @avekens)

Reply:
You are right I should obviously change "scalar product" into "[scalar multiplication](https://en.wikipedia.org/wiki/Scalar_multiplication)".

"Generalized Euclidean space" is what the definition [`df-rrx`](https://us.metamath.org/mpeuni/df-rrx.html) names that construct. Other nearby theorems also use that name, I just copied from them. Maybe we could open a different issue if this needs renaming?

Reply:
Maybe open an issue.
After looking at df-rrx I think the best name for `` ( RR^ ` I ) `` is simply "`` ( RR^ ` I ) ``". It is a specific space so name it by its name.

Reply:
> Maybe open an issue. After looking at df-rrx I think the best name for `` ( RR^ ` I ) `` is simply "`` ( RR^ ` I ) ``". It is a specific space so name it by its name.

I think "Generalized Euclidean space" is an adequate pronounceable name for this class. `` ( RR^ ` I ) `` is not a name, in my opinion. How would you pronounce it ("arar caret at i"?).

Reply:
It seems I'll have to write an issue.